opm/simulators/linalg/cuistl/CuOwnerOverlapCopy.hpp

```diff
@@ -141,9 +362,54 @@ class CuOwnerOverlapCopy
             }
         }
 
-        m_indicesCopy = std::make_unique<CuVector<int>>(indicesCopyOnCPU);
-        m_indicesOwner = std::make_unique<CuVector<int>>(indicesOwnerCPU);
+        this->m_indicesCopy = std::make_unique<CuVector<int>>(indicesCopyOnCPU);
+        this->m_indicesOwner = std::make_unique<CuVector<int>>(indicesOwnerCPU);
```

Comment:
I have removed them now from the base class, but for the derived classes it seems necessary when accessing variables from the base class. Not sure why.

Reply:
members inherited from a template base class requires this->  usage.

Reply:
The general trend in OPM code is to write all member variable access as this->member, consensus seems to be that this is a good idea. It may reduce the need to mark members by underscore postfix etc. as well. I have personally been a bit sceptical about making this a rule or strong recommendation, not least because it is not easy to check that it is followed 100% in any given class, but I have no strong objections.